app/services/fastly_config/update.rb

```diff
@@ -0,0 +1,58 @@
+module FastlyConfig
+  # Handles updates to our Fastly configurations
+  class Update
+    FASTLY_CONFIGS = %w[Snippets].freeze
+
+    class << self
```

Comment:
This `run` method has become pretty long. Maybe we should make this follow a pattern we use in other services, i.e.

```ruby
...
    def self.call
      new.run
    end

    attr_reader :service

    def initialize
        fastly = Fastly.new(api_key: ApplicationConfig["FASTLY_API_KEY"])
        @service = fastly.get_service(ApplicationConfig["FASTLY_SERVICE_ID"])
       ...
    end

    def run(configs: FASTLY_CONFIGS)
      ...
    end
```